packages/compass-indexes/src/components/create-index-form/index-flow-section.tsx

```diff
@@ -0,0 +1,183 @@
+import {
+  Body,
+  Button,
+  css,
+  cx,
+  Icon,
+  Label,
+  Link,
+  palette,
+  spacing,
+  Toggle,
+  Tooltip,
+} from '@mongodb-js/compass-components';
+import React from 'react';
+
+const flexContainerStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const indexFieldsHeaderContainerStyles = css({
+  justifyContent: 'space-between',
+  marginBottom: spacing[200],
+});
+
+const indexFieldsCalloutStyles = css({
+  border: `1px solid ${palette.gray.light2}`,
+  borderRadius: '12px',
+  padding: spacing[600],
+  marginBottom: spacing[600],
+});
+
+const codeEquivalentToggleLabelStyles = css({
+  marginRight: spacing[100],
+  fontWeight: 'normal',
+});
+
+const coveredQueriesHeaderContainerStyles = css({
+  marginBottom: spacing[200],
+});
+
+const coveredQueriesCalloutStyles = css({
+  border: `1px solid ${palette.gray.light2}`,
+  background: palette.gray.light3,
+  borderRadius: '12px',
+  padding: spacing[600],
+  marginBottom: spacing[600],
+});
+
+const buttonContainerStyles = css({
+  display: 'flex',
+  justifyContent: 'right',
+});
+
+const coveredQueriesButtonStyles = css({
+  height: spacing[600] + 4,
+  float: 'right',
+  marginTop: spacing[400],
+});
+
+const codeStyles = css({
+  fontFamily: 'Source Code Pro',
```

Comment:
We should use the fonts from LeafyGreen here (it is Source Code Pro, but it has some other fallbacks, and it'll make it so if anything changes it also gets changed here).

```ts
import { fontFamilies } from '@leafygreen-ui/tokens';
// fontFamily: fontFamilies.code
```